test/joystick_test.py

```diff
@@ -31,21 +37,38 @@ def todo_test_Joystick(self):
 
 
 class JoystickModuleTest(unittest.TestCase):
-    def todo_test_get_count(self):
-
-        # __doc__ (as of 2008-08-02) for pygame.joystick.get_count:
+    @unittest.skipIf(0 == JOYSTICK_COUNT, "No joysticks detected")
+    def test_get_count(self):
+        # Test get_count correctly identifies number of connected joysticks
+        prompt("Please add or remove joysticks as necessary before the test.")
```

Comment:
For this line it would probably be clearer for a modern audience if we went with something like:

> "Please connect any joysticks/controllers now before starting the joystick.get_count() test."

That tells people which test is happening and also gives some indication they can connect console controllers and not just an old Thrustmaster 7000 from the loft or something.